sdcm/stress_thread.py

```diff
@@ -343,11 +350,17 @@ def run(self):
         for loader_idx, loader in enumerate(self.loaders):
             for cpu_idx in range(self.stress_num):
                 for ks_idx in range(1, self.keyspace_num + 1):
+                    _wait_for_schema = CassandraStressThread.should_wait_for_schema
+                    if CassandraStressThread.should_wait_for_schema and self.prometheus_db:
+                        self.schema_created.wait()
+                    else:
+                        CassandraStressThread.should_wait_for_schema = True
+
                     self.results_futures += [self.executor.submit(self._run_cs_stress,
                                                                   *(loader, loader_idx, cpu_idx, ks_idx))]
-                    if loader_idx == 0 and cpu_idx == 0 and self.max_workers > 1:
-                        # Wait for first stress thread to create the schema, before spawning new stress threads
-                        time.sleep(30)
+                    if _wait_for_schema and self.prometheus_db:
+                        self.prometheus_db.metric_has_data(ANY_CS_METEIC_QUERY, n=10)
+                        self.schema_created.set()
```

Comment:
Now that we have consistent_schema_management, I don't think we should do it.
The main reasons are:
1. It's a good place where we actually test this part of the code.
2. It can create us a problem, especially in performance tests where we want all stress commands to run at the same time.
With this fix, I'm afraid that the first stress will run few minutes before the others and our stats in the beginning and in the end of the tests will be less accurate.

Reply:
1. we seen issues even with `consistent_schema_management: false`, once we have a fix we can remove.

2. regardless, this option is better then the 30sec sleep we have now.
    removing the sleep would create issues when using multiplier 

this implementation has one assumption, that there is only one schema we care about, i.e. if there would be multiple keyspace/tables, it would work only on the first one.